README.md

```diff
@@ -16,6 +17,12 @@ This route supports signup to several types of content. It was [originally speci
 
 Each of these routes corresponds to a content item with an [`email_alert_signup` schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/email_alert_signup.jsonnet). At the time of writing, `/foreign-travel-advice/*` and `/service-manual/*` still use this route.
 
+- A `/email/subscriptions/new` route [[example](https://www.gov.uk/email/subscriptions/new?topic_id=statistics-with-1-research-and-statistic-5e2982632b)].
+
+This route enables any other application to offer a fully customised email signup experience that is not reliant on the content store.
+
+It is used by [finder-frontend](https://github.com/alphagov/finder-frontend), where a new subscriber list is created from the combination of selected filters.
```

Comment:
```suggestion
It is used by other apps, like [finder-frontend](https://github.com/alphagov/finder-frontend), where a new subscriber list is created from the combination of selected filters.
```

Another example of this is the Brexit Checker, but I don't think we need to list them all.

Reply:
Updated